packages/react/radio-group/src/RadioGroup.tsx

```diff
@@ -95,9 +81,18 @@ const RadioGroup = React.forwardRef((props, forwardedRef) => {
   );
 
   return (
-    <RadioGroupContext.Provider value={context}>
+    <RadioGroupProvider
+      name={name}
+      value={value}
+      required={required}
+      rovingFocus={rovingFocus}
+      onValueChange={React.useCallback(
+        composeEventHandlers(handleValueChange, (event) => setValue(event.target.value)),
+        [handleValueChange]
+      )}
```

Comment:
This made me realise that we're passing the `event` to the consumer which doesn't seem to be inline with how we handle things elsewhere. I was expecting it to be `onValueChange(value: string): void;`. 

The docs say it has that signature but it actually doesn't 😬 

![image](https://user-images.githubusercontent.com/175330/109200867-00ad6f80-7799-11eb-851f-f5de667b86f3.png)

Shall I change it while I'm here?

Reply:
hmm weird! Let's tackle separately.